templates/component/src/index.js

```diff
@@ -0,0 +1,30 @@
+import { tag } from '@storefront/core';
+
+@tag('<%= slug %>', require('./<%= slug %>.html'), require('./<%= slug %>.scss'))
+export class <%= sanitizedName %> {
+  onBeforeMount() {
+    console.log('__ ...'); // TEMP
+  }
+
+  onMount() {
+    console.log('__ ...'); // TEMP
+  }
+
+  onUpdate() {
```

Comment:
Consider adding the (SF-specific) parameters for `onUpdate`. A few of the other lifecycle methods also receive parameters.

Reply:
implemented, resolving.